march_rqt_gait_generator/src/march_rqt_gait_generator/gait_generator_view.py

```diff
@@ -36,6 +37,12 @@ def __init__(self):
 
         self.gait_type_combo_box.addItems(['walk_like', 'sit_like', 'stairs_like'])
 
+        self.initialize_shortcuts(self)
+
+    def initialize_shortcuts(self, parent):
+        self.ctrl_z = QShortcut(QKeySequence(self.tr('Ctrl+Z')), parent)
+        self.ctrl_shift_z = QShortcut(QKeySequence(self.tr('Ctrl+Shift+Z')), parent)
```

Comment:
this method has a `parent` argument, but you never call it with that argument. Why is that?

Reply:
Woops that is an artefact of something I tried that did not work :sweat_smile: